pkg/apis/rollouts/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
code-gen failing for this line. please remove this line

Reply:
Thank you; undone.
Does the `code-gen` has an issue running on Mac?